### ISSUE TITLE
K33 support

### ISSUE THREAD

**Issue by rijulluman** [madMAx43v3r/chia-plotter#416]
Hello
Is it possible to get K33 and K34 support?
Since the plotting timings are only 1 hr now for most people, I think people may prefer plotting larger sized plots in 2-3hrs...

**Comment by rijulluman** [comment]
If its somehow possible to do the K33 plotting with the same 110gb RAM drive, that would be best case for larger farmers...
But I don't think that's possible?

**Comment by Dartellum** [comment]
K33 support was asked numerous times already. Please use the search function to see if your question/issue was asked already. 

The short answer, yes K33 will get added at some point in the future, most-likely when it is required.

**Comment by lordzamura** [comment]
i think k33 is not require for this time. check blockes. %95 of it is k32

**Comment by rijulluman** [comment]
@madMAx43v3r Sorry for the repetitive K33 request...
I just read your answer on https://github.com/madMAx43v3r/chia-plotter/issues/25 
`Once they announce k33 as minimum, I'll implement it, until then it's too much work.`
Could you maybe mention that in the Readme file?

**Comment by gavinmonson** [comment]
Hi,

I know k33 34 35 etc isnt coming but I'd like to add an additional minor argument for it's implimentation one day.

On a 1tb drive I have 9x k32 plots but I could also have 1xk35 not much difference in terms of winning but the k35 will pass the plot filter 9 times less often.

The 1 in 512 plot filter and 64 signage points per 10 mins gives a (1/512 x 64 x 100% = 12.5%) 12.5% chance of a plot being accessed every 10 minutes.

This means with the k32 plotted drive will be accessed on average every 8 mins while the k35 plotted drive will be accessed every 80 mins.

My point being that I could happily allow 1w standby on the k35 drive knowing will spin up less than once an hour saving quite a bit of power, around 80%.  Meanwhile on the k32 plotted drive spinning up and down or unloading the heads every 8 mins would wear out the drive too quickly so I have to run it full power all the time.

Basically higher k numbers take less power to farm is my point and this might be important to people running large numbers of drives.

Food for thought.
